File: Assgn1/Qn2/q2.c
#include<stdio.h>
#include<math.h>
int main() {
  double a = 0.2,b;
  double result[1000];
  result[0] = 0.2;
  for (int i = 1;i<1000;i++) {
    b = (a + M_PI)*100 - (int)((a + M_PI)*100);
    a = b;
    result[i] = a;
  }

  for (int i=0;i<1000;i++) {
    printf("%d: %.4f\n",i+1,result[i]);
  }
  return 0;
}

File: Assgn1/Qn2/q2.py
import math

array = []
result = []
array.append(0.2)
result.append(0.2)

for i in range(1,1000):
    array.append(((array[i-1] + math.pi)*100) - (int)((array[i-1] + math.pi)*100))
    result.append('%.4f' %array[i])

for i in range(0,len(result)):
    print '%d: %s' %(i+1, result[i])
